experimental/integrations/marquez-spark-agent/integrations/sparkrdd/1.json

```diff
@@ -10,8 +10,8 @@
     "name" : "job_name"
   },
   "inputs" : [ {
-    "namespace" : "ns_name",
-    "name" : "data.txt"
+    "namespace" : "gs.bucket",
+    "name" : "gs://bucket/data.txt"
```

Comment:
A dataset also has a `type` we'll need to set, will that be parsed from the `namespace` or `name`?

Reply:
The type is the scheme part of the URI